File: Java-OOP/01. Working with Abstraction - Lab/src/StudentSystem/Student.java
package StudentSystem;

public class Student {
    private String name;
    private Integer age;
    private Double grade;

    private String message;

    public String GetMessage() {
        return message;
    }

    public void SetMessage(String message) {
        this.message = message;
    }

    public String GetName() {
        return name;
    }

    public Integer GetAge() {
        return age;
    }

    public Double GetGrade() {
        return grade;
    }

    public Student(String command, String name, Integer age, Double grade) {
        if (command.toLowerCase().equals("create")) {
            this.name = name;
            this.age = age;
            this.grade = grade;
        }
    }

    public Student() {

    }

    public void DisplayInformation(Student student) {
        if (student.GetGrade() >= 5.50) {
            student.SetMessage("Excellent student");
        }
        else if (student.GetGrade() >= 3.00) {
            student.SetMessage("Average student");
        }
        else {
            student.SetMessage("Very nice person");
        }

        System.out.printf("%s is %d years old. %s.\n" , student.GetName(), student.GetAge(), student.GetMessage());
    }
}

File: Software Engineering/CSharp-Advanced/04. Multidimensional Arrays - Exercise/test/Student.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace test
{
    internal class Student
    {
        public string? Name { get; set; }
        public int Age { get; set; }
        public double Grade { get; set; }
        public string? Message { get; set; }

        public Student(string? name, int age, double grade)
        {
            Name = name;
            Age = age;
            Grade = grade;
        }

        public void DisplayInformation(Student student)
        {
            if (student.Grade >= 5.50)
            {
                student.Message = "Excelent student";
            }
            else if (student.Grade >= 3.00)
            {
                student.Message = "Average student";
            }
            else
            {
                student.Message = "Very nice person";
            }

            Console.WriteLine($"{student.Name} is {student.Age} years old. {student.Message}.");
        }
    }
}
